plugin/db/util/driverutil.go

```diff
@@ -377,7 +377,7 @@ func ExecuteMigration(ctx context.Context, l *zap.Logger, dbType db.Type, driver
 			}
 		}
 		// MySQL executes DDL in its own transaction, so there is no need to supply a transaction.
-		if err = driver.Execute(ctx, statement); err != nil {
+		if err = driver.Execute(ctx, statement, !m.CreateDatabase); err != nil {
```

Comment:
Adding a comment for postgres? And move the comment in Execute here?

Postgres doesn't allow creating database inside a transaction https://github.com/bytebase/bytebase/issues/202

Reply:
We'd better retain the existing mysql comment as well, since they are not identical? MySQL won't fail in this case